src/common/components/nav/navCartIcon.tpl.html

```diff
@@ -10,7 +10,8 @@
       class="nav-cart-icon nav-cart-icon-desktop"
       ng-click="$ctrl.miniCartViewAnalyticsEvent()"
     >
-      <span class="fal fa-shopping-cart">
+      <span class="material-icons-outlined">
+        volunteer_activism
         <span class="cart-item-count" ng-if="!$ctrl.cartOpen"></span>
```

Comment:
Looks like the Figma has restyled the count bubble to be blue. It seems to be blue on dev-cloud.cru.org, but not on give-dev-cloud.cru.org